src/components/report/FontSummary.vue

```diff
@@ -58,14 +58,14 @@
 			</strong>
 		</p>
 
-		<table class="details">
-			<tbody>
-				<tr v-for="(value, key) in summary" :key="key">
-					<th scope="row">{{ key }}</th>
-					<td v-html="value" />
-				</tr>
-			</tbody>
-		</table>
+		<dl class="details">
+			<!-- Disabling this Vue v2 rule here: https://eslint.vuejs.org/rules/no-v-for-template-key.html ->
+			<!-- eslint-disable-next-line vue/no-v-for-template-key -->
```

Comment:
Ah too bad. We could wrap the `dt` and `dd` together in a `div`, that is legal HTML. I would love for these comments to not make it to the frontend code: 

<img width="792" height="619" alt="image" src="https://github.com/user-attachments/assets/6a7e3ecb-ab8f-43c7-969f-999fbef4d62a" />